lib/engines/content_block_manager/app/components/content_block_manager/shared/embedded_objects/summary_card_component.rb

```diff
@@ -16,10 +16,18 @@ def title
   end
 
   def rows
+    object_keys_rows
+  end
+
+  def object_keys_rows
     object.keys.map do |key|
       {
         key: key.titleize,
         value: object[key],
```

Comment:
I think we need to have the embed code outputted here too, then hidden when we initialize the `copy-embed-code` JS module.

Reply:
ah yeah I meant to say I'd pick the no-JS option up in another PR!

Reply:
as the new design is a bit different and I think will mean changes to the actual javascript https://trello.com/c/Hr1mmk1C/899-add-copy-embed-code-to-attributes?filter=label:Dev